Examples/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(PcapPlusPlusExamples)
```

Comment:
I noticed that when building with Visual Studio there is a solution files (`.sln`) for:
- Everything - libs, tests, examples, etc.
- Just the examples
- Just the tutorials
- A few 3td party libs

![image](https://user-images.githubusercontent.com/9059541/187603954-edfe180a-1e70-4fff-95a0-c5978443a869.png)

The reason is that CMake generates a `.sln` file anywhere it finds a `project` command as explained here: https://cognitivewaves.wordpress.com/cmake-and-visual-studio/

Why do we need a `project` command here and in `Tutorials`?
If we remove them only one `.sln` file will be generated: `PcapPlusPlus.sln`, and the users can decide if they want to build tests, examples and tutorial using the `PCAPPP_BUILD_TESTS` and `PCAPPP_BUILD_EXAMPLES` flags

Reply:
I removed the tests project as test should be build inside the project but keep the examples to allow building when Pcap++ is already present on the computer